### PR TITLE
use same version for library and compiler

### LINKED ISSUE #190: Use same compose version for library and compiler
## Kind (Required)

- Improvement

 ## Overview (Required)

- Maybe you can declare the version in Dep.kt and use it throughout the app.

 ## Links	

- https://github.com/DroidKaigi/conference-app-2021/blob/main/buildSrc/src/main/java/Dep.kt#L7-L16
- https://github.com/DroidKaigi/conference-app-2021/blob/0f42f6898c2be402e101157b490513da06ce1ca8/build.gradle#L5
- https://github.com/DroidKaigi/conference-app-2021/blob/d5f412259a82de8c829adab12eb6f7dc24137b13/uicomponent-compose/feed/build.gradle#L30

### PR DESCRIPTION
**Pull request by risako070310**
## Issue
- close #190

## Overview (Required)
- use same version for library and compiler

## Links
-

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by mhidaka** [comment]
👀

File: buildSrc/src/main/java/Dep.kt [hunk 1]
```diff
@@ -1,20 +1,24 @@
...
-        const val iconsExtended = "androidx.compose.material:material-icons-extended:1.0.0-beta01"
-        const val animation = "androidx.compose.animation:animation:1.0.0-beta01"
+        const val runtime = "androidx.compose.runtime:runtime:${Versions.compose}"
+        const val livedata = "androidx.compose.runtime:runtime-livedata:${Versions.compose}"
+        const val foundation = "androidx.compose.foundation:foundation:${Versions.compose}"
+        const val layout = "androidx.compose.foundation:foundation-layout:${Versions.compose}"
+        const val ui = "androidx.compose.ui:ui:${Versions.compose}"
+        const val tooling = "androidx.compose.ui:ui-tooling:${Versions.compose}"
+        const val material = "androidx.compose.material:material:${Versions.compose}1"
+        const val iconsExtended = "androidx.compose.material:material-icons-extended:${Versions.compose}"
```
Comment (punchdrunker) [inline buildSrc/src/main/java/Dep.kt:20]: I believe putting a new line after `=` fix the warning by reviewdog

**Review by mhidaka — APPROVED** [review]
That's great! 💯

File: buildSrc/src/main/java/Dep.kt (hunk 1 above)
Comment (mhidaka) [reply buildSrc/src/main/java/Dep.kt:20]: @risako070310 Could you apply ktlint for a line ?
```
Step 'ktlint' found problem in 'buildSrc/src/main/java/Dep.kt':

> Task :spotlessKotlin FAILED
Error on line: 20, column: 1
Exceeded max line length (100)
java.lang.AssertionError: Error on line: 20, column: 1
Exceeded max line length (100)
```

**Comment by mhidaka** [comment]
Thank you for your contrib!

**Pull request merged**